### PR TITLE
Docs: Alternative IIS Task Driver

### PR DESCRIPTION
**Pull request by sevensolutions**
I'am working on an alternative IIS task driver for a while.
If you find this usefull for the entire community, feel free to merge this PR to list it on the community-plugins page.

Source Repo: https://github.com/sevensolutions/nomad-iis

### CODE REVIEW

File: website/content/plugins/drivers/community/index.mdx [hunk 1]
```diff
@@ -32,6 +32,7 @@ Below is a list of community-supported task drivers you can use with Nomad:
 - [Singularity][singularity]
 - [systemd-nspawn][nspawn-driver]
 - [Windows IIS][nomad-driver-iis]
+- [Windows IIS (Alternative)][nomad-iis]
```
Comment (jrasell) [inline website/content/plugins/drivers/community/index.mdx:35]:
```suggestion
- [Windows IIS][nomad-iis]
```

File: website/content/plugins/drivers/community/nomad-iis.mdx [hunk 2]
```diff
@@ -0,0 +1,75 @@
+---
+layout: docs
+page_title: 'Drivers: nomad-iis'
+description: >-
+  The IIS driver is used for running
+  Windows IIS services.
+---
+
+# Windows IIS Driver (Alternative)
```
Comment (jrasell) [inline website/content/plugins/drivers/community/nomad-iis.mdx:9]:
```suggestion
# Windows IIS Driver
```

File: website/data/docs-nav-data.json [hunk 3]
```diff
@@ -1931,6 +1931,10 @@
           {
             "title": "Windows IIS",
             "href": "/plugins/drivers/community/iis"
+          },
+          {
+            "title": "Windows IIS (Alternative)",
```
Comment (jrasell) [inline website/data/docs-nav-data.json:1936]:
```suggestion
            "title": "Windows IIS",
```

File: website/data/plugins-nav-data.json [hunk 4]
```diff
@@ -69,6 +69,10 @@
           {
             "title": "Windows IIS",
             "path": "drivers/community/iis"
+          },
+          {
+            "title": "Windows IIS (Alternative)",
```
Comment (jrasell) [inline website/data/plugins-nav-data.json:74]:
```suggestion
            "title": "Windows IIS",
```

File: website/content/plugins/drivers/community/nomad-iis.mdx [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+Name: `nomad_iis`
+
+This plugin is an alternative Task Driver to run IIS Web Applications on Windows Servers. Each allocation will create an AppPool and a Website.
```
Comment (jrasell) [inline website/content/plugins/drivers/community/nomad-iis.mdx:13]:
```suggestion
This plugin is a task driver to run IIS Web Applications on Windows Servers. Each allocation will create an AppPool and a Website.  
```

**Review by jrasell — CHANGES_REQUESTED** [review]
Hi @sevensolutions and thanks for raising this PR. I have suggested some changes which remove the term alternative, as this driver shouldn't be classed as such as it indicates we have a preference. Once this changes has been pushed, I'll get this merged.

**Review by jrasell — APPROVED** [review]
LGTM, thanks @sevensolutions!

**Comment by sevensolutions** [comment]
@jrasell do you have an idea, why this changes arent on the stable-website branch anymore?
They have been backported by #19693 and i know they were there, but now they are not there anymore but the files are still in the main branch.

This was the link:
https://developer.hashicorp.com/nomad/plugins/drivers/community/nomad-iis

Maybe theres an issue with the backporting pipeline?

**Pull request merged**